Common/Meshes/Mesh.cs

```diff
@@ -90,16 +94,17 @@ public void Update()
     {
         GL.BindVertexArray(VaoId);
         GL.BindBuffer(BufferTarget.ArrayBuffer, VboId);
-        GL.BufferData(BufferTarget.ArrayBuffer, Vertices.Length * Marshal.SizeOf<Vertex>(), IntPtr.Zero, BufferUsageHint.StaticDraw);
-        IntPtr ptr = GL.MapBuffer(BufferTarget.ArrayBuffer, BufferAccess.WriteOnly);
+        GL.BufferData(BufferTarget.ArrayBuffer, Vertices.Length * Marshal.SizeOf<Vertex>(), Vertices, BufferUsageHint.StaticDraw);
+        /*IntPtr ptr = GL.MapBuffer(BufferTarget.ArrayBuffer, BufferAccess.WriteOnly);
```

Comment:
delete commented code

Reply:
That's another thing to consider, how should we update the buffer? This second technique is called buffer orphaning and I read it could be faster, but I don't see any difference. A benchmark would probably be needed

Reply:
if you read its faster its fine with me. I does not seem to break anything